### ISSUE TITLE
Add test scripts for the different DL frameworks

### ISSUE THREAD

**Issue by CrohnEngineer** [polimi-ispl/blank_project#4]
It would be nice to have some test scripts written with the different deep learning frameworks we use (e.g., TF/Keras, PyTorch, PyTorch lightning, etc...)

**Comment by fpicetti** [comment]
Do you mean unit-tests?

**Comment by CrohnEngineer** [comment]
I mean a script for evaluating the networks on a test set.
I can take care of the PyTorch script, I would ask you or Francesco to do the Lightning part as I'm not really used to it (:  
For the unit-tests, it is a nice idea! Recently I've played with GitHub actions so we could also implement a CI pipeline for the repo.
However, I would list it as a separate TBD.

**Comment by fpicetti** [comment]
Ok, got it! I'll manage Lighting.
Not sure if unit-tests would be useful on this repo...

**Comment by fpicetti** [comment]
Done for pytorch and lightning in d41283abcf6b8b2b74c0babf7a10ddd92b7979fb
